src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/controller/GoalController.kt

```diff
@@ -49,7 +49,9 @@ class GoalController(private val service: GoalService) {
   fun deleteGoal(
     @PathVariable goalUuid: UUID,
   ) {
-    service.deleteGoal(goalUuid) ?: throw NoResourceFoundException(HttpMethod.DELETE, "No goal found for $goalUuid")
+    if (service.deleteGoalByUuid(goalUuid) != 1) {
```

Comment:
Can you name this magic number?

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/104/commits/35c138c3305c49a6e5495fbdba9018d327b2cba8